subsys/bluetooth/host/conn.c

```diff
@@ -130,6 +109,68 @@ static struct bt_conn sco_conns[CONFIG_BT_MAX_SCO_CONN];
 #endif /* CONFIG_BT_CLASSIC */
 #endif /* CONFIG_BT_CONN */
 
+#if defined(CONFIG_BT_CONN_TX)
+void frag_destroy(struct net_buf *buf);
+
+/* Storage for fragments (views) into the upper layers' PDUs. */
+/* TODO: remove user-data requirements */
```

Comment:
I assume you have a plan on doing that in a follow up PR or something? The TODO isn't really useful for other people, as the requirement or what to do be done is not written :)

Reply:
yeah it's a TODO for future me.
The user-data requirements I'm talking about is the use of `user_data` at all. Ownership of this field is confusing and error-prone so it'd be great if we didn't use it. I don't have a solution yet however.